activesupport/lib/active_support/parameter_filter.rb

```diff
@@ -28,12 +28,27 @@ module ActiveSupport
   class ParameterFilter
     FILTERED = "[FILTERED]" # :nodoc:
 
-    def initialize(filters = [])
+    # Create instance with given filters. Supported type of filter is +String+, +Regexp+, and +Proc+.
```

Comment:
Typo: 

`Supported type of filter is +String+, +Regexp+, and +Proc+.` ->
`Supported types of filters are  +String+, +Regexp+, and +Proc+.`